CONTRIBUTING.md

```diff
@@ -15,52 +55,221 @@ simple statement that you, as a contributor, have the legal right to make the
+  scheduled for work faster. They are also more accessible
+  to the community, meaning that you may not have to wait for the core team to get to it.
+1. Please do not open an issue with a description that is simply a link to another issue,
+  a link to a slack conversation, a quote from either one of those, or anything else
+  equally opaque. This raises the bar for entry and makes it hard for the community
+  to get involved. Take the time to write a proper description and summarise key points.
+1. Take care with formatting. Ensure the [markdown is tidy](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/getting-started-with-writing-and-formatting-on-github),
+  use [code blocks](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks) etc etc.
+  The faster something can be read, the faster it can be dealt with.
+1. Keep it civil. Yes, it is annoying when things don't work, but it is way more fun helping out
+  someone who is not... the worst. Remember that conversing via text exacerbates
+  everyone's negativity bias, so throw in some emoji when in doubt :+1: :smiley: :rocket: :tada:.
+
+**Dedicated guides**:
+- [Bug report guide](#bug-report-guide)
+- [Feature request guide](#feature-request-guide)
+- [Help request guide](#help-request-guide)
+
+## Bug report guide
+
+We hope to get to bug reports within a couple of working days, but please wait for at least
+7 before nudging. (Unless it is a super critical end-of-the world bug, then by all means
+make some noise :loudspeaker:.)
+
+Below are the criteria we like our bug reports to cover in order to gather the bare minimum of
+information. Add more that what is asked for if you can :smiley:.
+
+1. **Search existing issues.** If something similar already exists, and is still open, please contribute to the discussion there.
+
+1. **Write a concise and descriptive title**, like you would a commit message, something which others can easily
+  find when doing step 1 above.
+
+1. **Detail what it was that you were trying to do and what you expected would happen**.
+  Give some background information around what you have already done to your cluster, any custom configuration etc.
+  With sufficient information you can pre-empt any questions others may have. This should cut out some obvious
+  back-and-forth and help get people to the heart of the issue quickly.
+
+1. **Explain what actually happened**. Provide the relevant error message and key logs.
+
+1. **Provide a reproduction**, for example the exact command or a yaml file (sensitive info redacted).
+  Please try to reduce your reproduction to the minimal necessary to help whoever is helping you
+  get to the broken state without needing to recreate your entire environment.
+
+1. **Reproduce the issue with logging verbosity set to at least 4** (`-v=4`), if you have not already done so. Ensure
```

Comment:
Sometimes this is isn't feasible. It could be a bug they can’t reproduce easily or a command thats costly for them to re run. In a previous team I worked on when our CLI would exit non-zero we would dump the logs at max verbosity to a file (admittedly our CLI was mainly used in automation, so the noise associated with this wasn't as big an issue). wondering if something similar could potentially be useful?

Reply:
I think that would be a good feature to have, create a new issue?

I will re-word the instruction, add an "if possible" of something.

Reply:
Issue created https://github.com/weaveworks/eksctl/issues/3030